hadoop-ozone/dist/src/main/compose/ozonesecure-ha/.env

```
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 HDDS_VERSION=${hdds.version}
-HADOOP_VERSION=3
+HADOOP_VERSION=3.3.6
```

Now that the Docker image has the same version as Hadoop itself, we can change this to avoid the need for future updates:

```suggestion
HADOOP_VERSION=${hadoop.version}
```